science/vcftools/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            vcftools vcftools 0.1.16 v
+github.tarball_from     releases
+revision                0
+checksums               rmd160  2ddef86b8fa2c325bb063be0843e35655b83c932 \
+                        sha256  dbfc774383c106b85043daa2c42568816aa6a7b4e6abc965eeea6c47dde914e3 \
+                        size    480575
+
+categories              science
+platforms               darwin
+maintainers             nomaintainer
+license                 GPL-3
+
+description             utilities for working with VCF files
+
+long_description        A set of tools written in Perl and C++ for working with \
+                        VCF files, such as those generated by the 1000 Genomes \
+                        Project.
+
+homepage                https://vcftools.github.io/
+
+depends_lib             path:bin/perl:perl5
+
+variant universal {}
```

Comment:
Is this line needed? It should just work in a normal configure/make build.

Reply:
All these years, I still have no idea when this is needed and when it is not needed.... and the ```universal_variant no/yes``` business. No wonder new users have no clue either. The magical black-box features of MacPorts are both it's strength and greatest weakness.